src/plugins/unified_search/public/__stories__/search_bar.stories.tsx

```diff
@@ -711,4 +711,34 @@ storiesOf('SearchBar', module)
       submitButtonStyle: 'full',
       renderQueryInputAppend: () => <EuiButton onClick={() => {}}>Append</EuiButton>,
     })
+  )
+  .add('with quick filters', () =>
```

Comment:
This story is broken now! Can we fix it!

Reply:
Fixed, thanks for the heads up! 👍